PowerShell/JumpCloud Module/Tests/ModuleValidation/moduleValidation.Tests.ps1

```diff
@@ -97,6 +97,9 @@ Describe -Tag:('ModuleValidation') 'Module Manifest Tests' {
                 }
             }
             $todayDate = Get-Date -UFormat "%B %d, %Y"
+            if ($todayDate | Select-String -Pattern "0\d,") {
```

Comment:
If the date was something like November 01, 2023 this test would fail previously